### PR TITLE
KEYCLOAK-16918 Set custom user attribute to Name ID Format for a SAML client

### PR DESCRIPTION
**Pull request by i7a7467**
https://issues.redhat.com/projects/KEYCLOAK/issues/KEYCLOAK-16918
https://groups.google.com/g/keycloak-dev/c/4ExUdT9rc2g

### NameID Mapper description
I am worried where to add the Mapper description for the SAML client.
This time,I have added instructions only for the SAML client configuration part.

Code PR : https://github.com/keycloak/keycloak/pull/7913

### CODE REVIEW

File: server_admin/topics/clients/client-saml.adoc [hunk 1]
```diff
@@ -111,6 +111,28 @@ Name ID Format::
   Name ID Format for the subject.
   If no name ID policy is specified in the request or if the Force Name ID Format attribute is true, this value is used.
   Properties used for each of the respective formats are defined below. 
+  You can change the attribute used by the NameID Mapper.
+
+|===
+|Name ID Format|Properties/Attributes Used|Example
+
+|username(unspecified)
```
Comment (hmlnarik) [inline server_admin/topics/clients/client-saml.adoc]:
```suggestion
|username (unspecified)
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Could you please rebase?

**Comment by i7a7467** [comment]
@hmlnarik 
Thank you for your review comment.
I rebased and fixed your review point.

**Review by hmlnarik — APPROVED** [review]
@i7a7467 Thank you for the contribution!

**Comment by hmlnarik** [comment]
Build failures are unrelated to this PR.

**Pull request merged**